packages/components/src/Drawer/DrawerContent.tsx

```diff
@@ -0,0 +1,38 @@
+import type { CSSProperties, ReactNode } from 'react';
+import { cn, useClassNames } from '@marigold/system';
+import { useDrawerContext } from './Context';
+
+// Props
+// ---------------
+export interface DrawerContentProps {
+  variant?: string;
+  size?: string;
+  /**
+   * Children of the component.
+   */
+  children?: ReactNode;
+}
+
+// Component
+// ---------------
+export const DrawerContent = ({
+  variant,
+  size,
+  children,
+}: DrawerContentProps) => {
+  const ctx = useDrawerContext();
+  const classNames = useClassNames({
+    component: 'Drawer',
+    variant: variant ?? ctx.variant,
+    size: size ?? ctx.size,
+  });
+
+  return (
+    <div
+      className={cn('[grid-area:content]', classNames.content)}
+      style={{ '--i': 1 } as CSSProperties}
```

Comment:
what does `--i` stand for? Should it be more use case specific?

Reply:
i = index, what would your proposal be? this is only required for the animation stuff

Reply:
call it index xD but maybe its just me